bin/util.py

```diff
@@ -568,9 +568,14 @@ def substitute_dir_variables(dirname, variables):
 
 # copies a directory recursively and substitutes {%key%} by their value in text files
 # reference: https://docs.python.org/3/library/shutil.html#copytree-example
-def copytree_and_substitute(src, dst, variables, exist_ok=True, *, preserve_symlinks=True):
+def copytree_and_substitute(src, dst, variables, exist_ok=True, *, preserve_symlinks=True, base=None):
```

Comment:
You don't modify any call site, so can you not just remove `base` and replace it by `src` everywhere?

Reply:
fixed that. In the recursive call `base` should be the top level `src`